blocks/model.py

```diff
@@ -190,7 +179,13 @@ def get_params(self):
         parameters that do not belong to any brick.
 
         """
-        return self.params
+        if (len(set(parameter.name for parameter in self.parameters)) !=
+                len(self.parameters)):
+            raise ValueError("parameters should have different names")
```

Comment:
@bouthilx ,@rizar,
But the name of parameters may be the same. like weights are all ```w```'s.